WebRoot/WEB-INF/tags/img.tag

```diff
@@ -30,7 +30,7 @@
 <c:if test="${not empty altkey}"><fmt:message key="${altkey}" var="alt"/></c:if>
 <c:if test="${not empty title and not rawtitle}"><fmt:message key="${title}" var="title"/></c:if> 
 <c:if test="${zm:boolean(disabled)}"><c:set var="clazz" value="${clazz} ImgDisabled"/></c:if>
-<app:imginfo var="info" value="${src}" />
+<app:imginfo var="info" value="${fn:escapeXml(src)}" />
```

Comment:
wasn't this fixed recently by bhavin?

Reply:
Removed. This is also removed by Bhavin side.